opentok/opentok.py

```diff
@@ -559,7 +562,8 @@ def start_archive(
             "hasVideo": has_video,
             "outputMode": output_mode.value,
             "resolution": resolution,
-            "streamMode": stream_mode.value
+            "streamMode": stream_mode.value,
+            "multiArchiveTag": multi_archive_tag
```

Comment:
Please mimic the docs for the REST API docs: https://tokbox.com/developer/rest/#start_broadcast. Also, please add the docs for the multiiBroadcastTag property (and its implementation?), to the opentok.broadcast module. Also, docs comments for some of the other properties of the broadcast module are missing.